challenges/objects-arrays.js

```diff
@@ -50,6 +78,13 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+
+graduates.map((grads,i) => {
```

Comment:
Try and be a bit more selective with which array method you're using.  Map will return an array and in this instance would be a good choice, but instead of assigning that returned array to `universities` you're just looping over the `graduates` array and setting the index of the `universities` array to the graduate's individual element's university.  In that case, `forEach()` would be a better choice

Reply:
I think the optimal solution would be:
```
universities = graduates.map(element => element.university)
universities = universities.sort()
```

Reply:
you can also one line it by chaining the `.sort()` to the end of the map function:
```
universities = graduates.map(element => element.university).sort()
```

Reply:
I think at one point I tried both of those ways. But I kept getting `Uncaught TypeError: Assignment to constant variable.`

Would it have been acceptable to change `const` to `let` to get this to work? I was trying to get it to work without changing it.

Reply:
I didn't see the const!  That's a very good point.  I'd say `let` is acceptable.  Another option is to just do
```
const universities = graduates.map(element => element.university).sort()
```

Reply:
That way you're only declaring the variable once, but yes if it's a const, you can't reassign it but you can push/add to it.

Reply:
> I didn't see the const! That's a very good point. I'd say `let` is acceptable. Another option is to just do
> 
> ```
> const universities = graduates.map(element => element.university).sort()
> ```

I acutally used this format for the last request. 

```const populationTotal = zooAnimals.reduce((total, obj) => total + obj.population, 0);```